src/applications/facility-locator/containers/FacilitesMap.jsx

```diff
@@ -216,9 +226,21 @@ const FacilitiesMap = props => {
   if (props.results.length > 0 && map) {
     // Set dragend to track map-moved ga event
     map.on('dragend', () => {
-      recordPanEvent(map.getCenter(), props.currentQuery.searchCoords);
+      const searchAreaControlId = document.getElementById(
+        'search-area-control-container',
+      );
+
+      if (searchAreaControlId.style.display === 'none') {
+        searchAreaControlId.style.display = 'block';
+      }
+
+      if (searchAreaControlId && !searchAreaSet) {
+        searchAreaControlId.addEventListener('click', handleSearchArea, false);
+        searchAreaSet = true;
+      }
```

Comment:
A more conventional solution would be to use a React component as a child of the map, with a prop that controls whether it renders.

Reply:
https://codepen.io/sergei-zelinsky/pen/gGOZjE